src/Artsy/Router/Components/StorybookRelay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { QueryRenderer } from "react-relay"
+import { Environment, RecordSource, Store } from "relay-runtime"
+import { createMockNetworkLayer } from "../../Relay/createMockNetworkLayer"
+
+export const StorybookRelay = ({
+  Component,
+  query,
+  mockResolvers,
+}: {
+  Component: React.ComponentClass | React.SFC
```

Comment:
These two can be replaced by `React.ComponentType`

Reply:
If this unnamed interface were exported it would automatically generate a `.d.ts` file on build that could be read by consumers of Reaction.